h/static/scripts/streamer.js

```diff
@@ -69,6 +64,26 @@ function connect($websocket, annotationMapper, groups) {
         annotationMapper.unloadAnnotations(annotations);
         break;
     }
+  }
+
+  function handleSessionChangeNotification(message) {
+    session.update(message.model);
+  }
+
+  // Listen for updates
+  socket.onMessage(function (event) {
+    message = JSON.parse(event.data)
```

Comment:
This should probably be wrapped in a `try {  } catch { }`. I see it wasn't before, but invalid JSON over the websocket shouldn't explode the frontend. Happy to see this in a future PR instead of here.